modules/editable-layers/src/utils/geojson-types.ts

```diff
@@ -4,7 +4,17 @@
 
 // Geometry types
 
-export type Position = [number, number] | [number, number, number];
+import {
```

Comment:
Any reason why we need to alias names? Can't we just re-export "as is" from the geojson module?